### PR TITLE
[vcpkg] improve pull request template

### PR DESCRIPTION
**Pull request by autoantwort**
**Describe the pull request**

- What does your PR fix?
  Imho the current template is sometimes hard to read because the answers are behind the question.  
  I have also added the question if the person has executed `./vcpkg x-add-version --all` because a lot of people misses that.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
-- Which triplets are supported/not supported? Have you updated the CI baseline?
+- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
+  <all / linux, windows, ...>, <Yes/No>
 
-- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
+- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
+  `Your answer`
+
+- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
+  <Yes / I am still working on this PR>
```
Comment (ras0219-msft) [inline .github/pull_request_template.md:13]:
```suggestion
  <Yes / I am still working on this PR>

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**
```

**Review by ras0219-msft — APPROVED** [review]

**Comment by autoantwort** [comment]
The download of cmake failed, can you rerun the pipeline? :)

**Review by ytbigvinny27 — APPROVED** [review]

**Comment by autoantwort** [comment]
Currently blocked by the pipelines failures #16990, #17057, #17039 and #17005. 

Can you theoretically merge this even if the pipeline fails? Because I am pretty sure that the failures are not caused by updating the pull request template (or I really really hope so)

**Review by JackBoosY — APPROVED** [review]

**Comment by JackBoosY** [comment]
LGTM.

**Comment by PhoebeHui** [comment]
The CI failures has gone after rerun. They are unrelate to this changes. @autoantwort, thanks for the PR!

**Review by PhoebeHui — APPROVED** [review]

**Comment by autoantwort** [comment]
> Marking requires:discussion so we can go over the wording tomorrow just to make sure we don't have anything we want changed slightly.

What was the result. Should I change something? :)

**Comment by JackBoosY** [comment]
Nope, we'll have a discuss.

**Pull request merged**